tests/api/plugins/graphql/crud.test.api.js

```diff
@@ -346,34 +346,6 @@ describe('Test Graphql API End to End', () => {
       });
     });
 
-    test.skip('List posts with `createdBy` and `updatedBy`', async () => {
```

Comment:
Removed because those fields aren't accessible via graphql api

Reply:
if you enable the populateCreatorFields options they should be I believe 🤔

Reply:
ahh, I'll see if I can keep and fix that one then

Reply:
Ok, I checked and apparently it's not available in Graphql: https://docs.strapi.io/dev-docs/api/rest/guides/populate-creator-fields

Reply:
I don't think we're excluding them anywhere in the GQL code though. Did you try to manually enable the "populate creator fields" and check the GQL definitions?

Reply:
Yeah, I added populateCreatorFields to the content type in the tests and it didn't add them. So I checked the docs and that's what it said.

I can add a task to add the feature back because it does seem odd we don't have it.

Reply:
I added a `test.todo` and created an item in the DX backlog to add (back) the feature

Reply:
If it was documented, it definitely means it was intentionnal. now we need to remember why 🤔